### PR TITLE
fix: url 404

### PR DESCRIPTION
**Pull request by jljsj33**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered docs/spec/transition.zh-CN.md](https://github.com/ant-design/ant-design/blob/jljsj33-patch-1/docs/spec/transition.zh-CN.md)

### CODE REVIEW

**Comment by afc163** [comment]
还有英文。

File: docs/spec/transition.zh-CN.md [hunk 1]
```diff
@@ -70,4 +70,4 @@ title: 巧用过渡
 
 ## 自然运动
 
-参见 [Ant Motion 动画语言](http://motion.ant.design/language/time)。
+参见 [Ant Motion 动画语言](https://motion.ant.design/language/basic)。
```
Comment (afc163) [inline docs/spec/transition.zh-CN.md]: 应该是 https://motion.ant.design/language/basic-cn

Comment (jljsj33) [reply docs/spec/transition.zh-CN.md]: 额

**Comment by jljsj33** [comment]
好了

**Pull request merged**